src/index.html

```diff
@@ -5,8 +5,323 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
Replace the stylesheet link to main.scss with a link to main.css. Browsers don't understand SCSS directly, so it needs to be compiled to CSS first.